api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -408,6 +408,25 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		assertEquals(1, encountersByVisit.size());
 	}
 	
+	/**
```

Comment:
please remove these javadocs from all your tests they dont add more information and just take up space and need to be maintained. we know which method you test from the test method name and the method call in the test.

Reply:
@teleivo I am unable to find the above javadoc :(